pipert2/core/base/pipe.py

```diff
@@ -102,15 +103,19 @@ def build(self):
 
         self.event_board.build()
 
-    def notify_event(self, event_name: str, **event_parameters) -> None:
+    def notify_event(self, event_name: str, specific_flows_to_routines: dict = defaultdict(list), **event_parameters) -> None:
         """Notify an event has started
 
         Args:
             event_name: The name of the event to notify
+            specific_flows_to_routines: For notify specific routines/flows we insert a dictionary in this format -
+            For specific routines in specific flow, each key/value element need to be in this format - "flow_name": [routines]
+            For all routines in specific flow, each element need to be in this format - "flow_name" - []
+
```

Comment:
```suggestion
            specific_flow_routines: In order to notify specific routines/flows we insert a dictionary in the following format -
            For specific routines in a specific flow, each key/value element needs to be in this format - "flow_name": [routines]
            For all of the routines in a specific flow, each element needs to be in this format - "flow_name" - []

```

Remember to change the parameter name